docker/jenkins/common/install_nccl.sh

```diff
@@ -19,6 +19,8 @@ elif [[ "$UBUNTU_VERSION" == 14.04 ]]; then
   NCCL_DEB='nvidia-machine-learning-repo-ubuntu1404_4.0-2_amd64.deb'
 fi
 
+apt update
+apt-get install wget
```

Comment:
curl is already installed, you can use `curl -LO <url>` to make it write the file.